types/sharedb/lib/client.d.ts

```diff
@@ -26,7 +26,7 @@ export class Connection extends ShareDB.TypedEmitter<ShareDB.ConnectionEventMap>
     get(collectionName: string, documentID: string): Doc;
     createFetchQuery<T = any>(collectionName: string, query: any, options?: {results?: Array<Doc<T>>} | null, callback?: (err: Error, results: Array<Doc<T>>) => void): Query<T>;
     createSubscribeQuery<T = any>(collectionName: string, query: any, options?: {results?: Array<Doc<T>>} | null, callback?: (err: Error, results: Array<Doc<T>>) => void): Query<T>;
-    fetchSnapshot(collection: string, id: string, version: number, callback: (error: Error, snapshot: ShareDB.Snapshot) => void): void;
+    fetchSnapshot(collection: string, id: string, version: number | null, callback: (error: Error, snapshot: ShareDB.Snapshot) => void): void;
```

Comment:
While we're here, can we please also make `fetchSnapshotByTimestamp()`'s `timestamp` argument [nullable](https://github.com/share/sharedb/blob/082e27b973531845bc42f9a8f27401eee063e0fc/test/client/snapshot-timestamp-request.js#L494-L495)?

Reply:
Sure! Thanks for reviewing.